css/buttons.css

```diff
@@ -0,0 +1,3 @@
+html[dir="rtl"] .btn {
```

Comment:
We should probably also add `.wp-block-button a` and `.wp-block-file .wp-block-file__button`